injected/entry-points/mozilla.js

```diff
@@ -1,96 +1,45 @@
 /**
- * @module Mozilla integration
+ * @module Mozilla MV2 main world integration
```

Comment:
Looks like the files are now unified, should we just merge the entry points?

Reply:
Follow up :). The less of these files the better imo.

Reply:
How about a symlink? So that extension still treats them as separate, but there's no duplication in C-S-S